charts/intel-gpu-resource-driver/values.yaml

```diff
@@ -37,45 +37,10 @@ controller:
     {}
     #node-role.kubernetes.io/control-plane: ""
   affinity: {}
-  containers:
```

Comment:
Line 20 in this file above still has `rbac` section making an impression that it is used. Is it used?

Reply:
that was overlooked. now I removed that completely